File: backend/models/artwork.py
from models.result import Result

class Artwork(Result):
    def __init__(self, name) -> None:
        super().__init__('artwork', name)
        self.year = None
        self.description = {}
        self.authorUri = {}
        self.authorName = None
        self.wikipedia_link = None
        self.museumName = []
        self.provenance = {}
        pass
    
    

File: frontend/src/@types/artwork.ts
export type Artwork = {
    name : string,
    type : string,
    uris : Map<string, string>,
    image : string,
    description : Map<string, string> | null,
    authorUri : Map<string, string> | null,
    authorName : string | null,
    year : string | null,
    wikipedia_link : string | null,
    museumName : string[] | null,
}